HWMgmt/OCPBaselineRedfishService.json

```diff
@@ -0,0 +1,571 @@
+                    "ActionInfo": "Mandatory"
+                },
+                "ReplaceCertificate": {}
+            }
+        },
+        "EthernetInterface": {
+            "ReadRequirement": "Mandatory",
+            "URIs": [
+                "/redfish/v1/Managers/{ManagerId}/EthernetInterfaces/{EthernetInterfaceId}"
+            ],
+            "PropertyRequirements": {
+                "MACAddress": {},
+                "SpeedMbps": {
+                    "ReadRequirement": "Supported"
+                },
+                "InterfaceEnabled": {},
+                "LinkStatus": {},
+                "Status": {
+                    "PropertyRequirements": {
+                        "Health": {},
+                        "State": {}
+                    }
+                },
+                "DHCPv4": {
+                    "ReadRequirement": "Recommended",
+                    "WriteRequirement": "Recommended",
+                    "PropertyRequirements": {
+                        "DHCPEnabled": {
+                            "ReadRequirement": "IfImplemented"
+                        }
+                    }
+                },
+                "DHCPv6": {
+                    "ReadRequirement": "Recommended",
+                    "WriteRequirement": "Recommended",
+                    "PropertyRequirements": {
+                        "OperatingMode": {
+                            "ReadRequirement": "IfImplemented"
+                        }
+                    }
+                },
+                "HostName": {
+                    "ReadRequirement": "Supported"
+                },
+                "FQDN": {
+                    "ReadRequirement": "Supported"
+                },
+                "NameServers": {},
+                "IPv4Addresses": {
+                    "Address": {},
+                    "AddressOrigin": {},
+                    "Gateway": {},
+                    "SubnetMask": {}
+                },
+                "IPv4StaticAddresses": {
+                    "ReadRequirement": "Recommended"
+                },
+                "IPv6AddressPolicyTable": {
+                    "ReadRequirement": "Recommended"
+                },
+                "IPv6StaticAddresses": {
+                    "ReadRequirement": "Recommended"
+                },
+                "IPv6StaticDefaultGateways": {
+                    "ReadRequirement": "Recommended"
+                },
+                "IPv6Addresses": {
+                    "ReadRequirement": "IfImplemented",
+                    "PropertyRequirements": {
+                        "Address": {},
+                        "PrefixLength": {},
+                        "AddressOrigin": {},
+                        "AddressState": {}
+                    }
+                },
+                "StaticNameServers": {
+                    "ReadRequirement": "Recommended"
+                }
+            }
+        },
+        "EventDestination": {
```

Comment:
Need EventDestinationCollection to show insertable/deletable. May also need to flag the resource itself as "Supported" in case the collection is empty when the service is being tested.

Reply:
We may need to just specify this in the profile notes - if the unit under test doesn't have a destination configured, we won't be able to validate this resource for the required properties.  May just become a testing requirement that they have to create a destination for the test to pass?

Reply:
We actually added `IfPopulated` to the spec for this purpose. The resource is required, but it may not be present at the time of test.

> For resource-level requirements, this value indicates that the resource is required, but may not be present (populated) in the service at all times.